in-progress/8757-fees/design.md

```diff
@@ -0,0 +1,189 @@
+# Testnet Fees
+
+|                      |                                                                    |
+| -------------------- | ------------------------------------------------------------------ |
+| Issue                | https://github.com/AztecProtocol/aztec-packages/issues/8757        |
+| Owners               | @just-mitch @LHerskind                                             |
+| Approvers            | @Maddiaa0 @PhilWindle @dbanks12 @nventuro @joeandrews @aminsammara |
+| Target Approval Date | 2024-10-24                                                         |
+
+## Mana and Costs
+
+Refer to the unit of work on L2 as "mana", analogous to gas on L1.
+
+A transaction incurs the following costs:
+1. Proposers/validators simulate the transaction
+2. Proposers publish pending blocks to L1
+3. Provers generate the avm proofs for the transaction
+4. Provers generate the rollup proofs for the blocks/epoch
+5. Provers verify the epoch proof on L1
+
+Some costs are independent of the computation or data of the transaction itself, specifically numbers 2, 4, and 5; regardless, they must be covered by the transaction.
+
+## Protocol Defined Constants
+
+- `overhead_mana_per_tx` - the overhead cost in mana for a transaction
+- `target_mana_per_block` - the amount of mana that an "average" block is expected to consume
+- `blobs_per_block` - the number of blobs per block that are compensated for in the base fee
```

Comment:
Actually this will get especially rolled into the base fee because we straight up calculate the current^ cost of the blobs in wei and roll that into the base fee.

^current as of the last `UNDERLYING_BASE_FEE_ORACLE_UPDATE_INTERVAL`